src/runner.rs

```diff
@@ -1247,7 +1167,7 @@ impl Interpreter {
 }
 
 /// Function execution context.
-struct FunctionContext {
+pub struct FunctionContext {
```

Comment:
Why does this need to be public?

Reply:
`FunctionContext` is part of the type signature for `Interpreter::with_stacks`.

Reply:
Made `FunctionContext` private in b76ad46.